google/cloud/bigquery/v2/minimal/internal/job_idempotency_policy.cc

```diff
@@ -13,16 +13,13 @@
 // limitations under the License.
 
 // Implementation of internal interface for Bigquery V2 Job resource.
-// Source: //depot/google3/google/cloud/bigquery/v2/job.proto
 
 #include "google/cloud/bigquery/v2/minimal/internal/job_idempotency_policy.h"
 
 namespace google {
 namespace cloud {
-namespace bigquery_v2_minimal {
-namespace internal {
-    
-}  // namespace internal
-}  // namespace bigquery_v2_minimal
+namespace bigquery_v2_minimal_internal {
```

Comment:
PS: We should probably be adding `GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN` here.

Reply:
Okay. I'll add it similar to the following:

https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigquery/internal/bigquery_read_connection_impl.h#L38

and

https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigquery/internal/bigquery_read_connection_impl.h#L104

Reply:
Followup question: I also notice the following in other header files towards the end of the namespace

namespace gcpcxxV1 = GOOGLE_CLOUD_CPP_NS;  // NOLINT(misc-unused-alias-decls)

Reference: https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigquery/internal/bigquery_read_connection_impl.h#L105

Shoudl the need to be added as well?

Reply:
No, those are for backwards compatiblity.

Reply:
Done.